### ISSUE TITLE
Can't build dotnet core 2.0 console apps after reinstall

### ISSUE THREAD

**Issue by danielzoeller** [dotnet/cli#7765]
## Steps to reproduce
I previously had dotnet 2.0 and VS 2017 installed and working fine.  I had to do a system restore on my surface pro and then VS 2017 stopped working.  I uninstalled VS 2017 and dotnet core.  Then reinstalled both but I specifically can't get dotnet core 2.0 console apps to build.  Strangely enough, asp.net core 2.0 apps seem to be working just fine.  Also, I can build console apps for dotnet 1.1 just fine as well.

Even a simple Hello world app created from the CLI doesn't seem to work:

$ dotnet new console
$ dotnet restore
$ dotnet run

## Expected  behavior
Console app should run.

## Actual behavior
Here is the error I get:

```
C:\Users\danzo\AppData\Local\Temp\.NETCoreApp,Version=v2.0.AssemblyAttributes.cs(4,20): error CS0400: The type or namespace name 'System' could not be found in the global namespace (are you missing an assembly reference?) [C:\Users\danzo\Source\repos\Hello\Hello.csproj]
obj\Debug\netcoreapp2.0\Hello.AssemblyInfo.cs(10,12): error CS0246: The type or namespace name 'System' could not be found (are you missing a using directive or an assembly reference?) [C:\Users\danzo\Source\repos\Hello\Hello.csproj]
obj\Debug\netcoreapp2.0\Hello.AssemblyInfo.cs(11,12): error CS0246: The type or namespace name 'System' could not be found (are you missing a using directive or an assembly reference?) 

[OMITTED ABOUT 30 SIMILAR ERRORS FOR BREVITY]

The build failed. Please fix the build errors and run again.
```

## Environment data
.NET Command Line Tools (2.0.0)

Product Information:
 Version:            2.0.0
 Commit SHA-1 hash:  cdcd1928c9

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.15063
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\2.0.0\

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.0
  Build    : e8b8861ac7faf042c87a5c2f9f2d04c98b69f28d

----------------------------------------------------
**Thank you for any help you can provide me!**

**Comment by livarcocc** [comment]
Can you try cleaning up that temp folder and doing a build again? Does it work if you build from the command line?

**Comment by danielzoeller** [comment]
Thanks for the quick reply.  Looks like I get the same output after trying your suggestion.

- Cleaned up my temp folder
- dotnet restore
- dotnet build

Same error :(

Any other ideas?

**Comment by livarcocc** [comment]
Can you run your build command with /bl:log.binlog to generate a binary log of your build and share that here?

**Comment by danielzoeller** [comment]
Attached.  Had to add .txt so that it would work on here.

[log.binlog.txt](https://github.com/dotnet/cli/files/1360229/log.binlog.txt)

**Comment by livarcocc** [comment]
Sorry for the delay in replying back, haven't had time to take a look at the log yet. Will try to see if anything comes up now.

**Comment by livarcocc** [comment]
Ok, here is what I think happened, for some reason, you nuget cache under `C:\Program Files\dotnet\sdk\NuGetFallbackFolder` seems to be broken. Can you please navigate to that location and check the contents of `C:\Program Files\dotnet\sdk\NuGetFallbackFolder\microsoft.netcore.app\2.0.0\ref\netcoreapp2.0` and check what you have in there? You should have a bunch of dlls in it.

I am not sure how the system restore works, but I wonder if it restore part of your offline cache, but not all of it, however, enough to fool the CLI into expanding it again.

Also, if you can share your project.assets.json for your failing projects, it might help me confirm my suspicions.

One way that you can fix yourself is to uninstall the 2.0 CLI and then delete the `C:\Program Files\dotnet\sdk\NuGetFallbackFolder` folder. We don't remove it during the uninstall, then just re-install the 2.0.0 CLI and check that the path I pasted above now has dlls in it.

**Comment by danielzoeller** [comment]
Looks like this works!  Thank you so much for your time and expertise!

For posterity here is exactly what I did:

1. Uninstalled the entire .net core 2.0 library
2. Deleted (renamed actually, just to be safe) the entire NugetFallbackFolder
3. Reinstalled .net core 2.0
4. Ran a restore command on my project
Then the build worked!

Thanks again,
Dan
